upgrades/3.x/README.adoc

```diff
@@ -1,6 +1,8 @@
 = Upgrade Notes
 
 ifdef::env-github[]
+include::3.12.0/README.adoc[leveloffset=+1]
```

Comment:
Not necessary in this PR, will be done with https://github.com/gravitee-io/release/pull/337

Reply:
ok, removed it